package.json

```diff
@@ -51,7 +50,7 @@
     "@material-ui/icons": "4.11.2",
     "@rehooks/component-size": "1.0.3",
     "@styled-system/prop-types": "5.1.5",
-    "apollo-server": "2.24.1",
+    "apollo-server-micro": "2.25.2",
```

Comment:
please excuse my ignorance - just wondering what `apollo-server-micro` doing that `apollo-server` wasn't able to do?

Reply:
Vercel/Next.js uses micro to front the api functions, so it is the preferred integration.

Reply:
ah right right sounds good